docs/book/codegen.md

```diff
@@ -75,6 +75,9 @@ following keys (unknown keys are ignored):
   this value is not provided, you will need to set it with the generator's
   `setOutputDirectory()` method before calling `generate()`.
 
+- `logger`: By default `\Psr\Log\NullLogger` is defined. See the
+  [Logging](#logging) section below for details.
```

Comment:
* The leading backslash for the classname can be omitted because it is only needed in namespaced code.
* The hint "below" should avoid because only in the current state of the document is the section below but nobody knows if that will be the case in the future. The link is enough. 

```suggestion
- `logger`: By default `Psr\Log\NullLogger` is defined. See the
  [Logging section](#logging) for details.
```